packages/patternfly-4/react-integration/demo-app-ts/src/components/demos/TableDemo/TableEditableDemo.tsx

```diff
@@ -56,20 +57,13 @@ export class TableEditableDemo extends React.Component<TableProps, { columns: (I
           cells: [
             {
               title: (value: string, rowIndex: number, cellIndex: number, props: any) => (
-                <React.Fragment>
-                  <div className="pf-c-inline-edit__value">{value}</div>
-                  <div className="pf-c-inline-edit__input">
-                    <TextInput
-                      value={props.editableValue !== undefined ? props.editableValue : value}
-                      isValid={props.isValid}
-                      type="text"
-                      onChange={(value, event) => {
-                        this.handleTextInputChange(value, event, rowIndex, cellIndex);
-                      }}
-                      aria-label="Row 1 cell 1 content" />
-                    <div className="pf-c-form__helper-text pf-m-error" aria-live="polite">{props.errorText}</div>
-                  </div>
-                </React.Fragment>
+                <EditableTextCell
+                  value={value}
+                  rowIndex={rowIndex}
+                  cellIndex={cellIndex}
+                  props={props}
+                  handleTextInputChange={this.handleTextInputChange}
+                  inputAriaLabel="Row 1 cell 1 content" />
```

Comment:
Can an editable cell be invalid, do we need an isValid prop?

Reply:
Yep! `isValid` is actually part of that props object, EditableTextCell component looks for this and updates the TextInput accordingly.